test/e2e/testnet/testnet.go

```diff
@@ -218,6 +218,7 @@ func (t *Testnet) CreateAccount(name string, tokens int64, txsimKeyringDir strin
 	err = t.genesis.AddAccount(genesis.Account{
 		PubKey:  pk,
 		Balance: tokens,
+		Name:    name,
 	})
```

Comment:
[note to self] this may be why it is preferable to use a struct constructor like `NewGenesisAccount` instead of creating a struct via `genesis.Account{}` in many files. That way the PR that added a new field to the struct can add it to the `NewGenesisAccount` constructor and doesn't have to CMD + F for all instances of the struct in the entire codebase to make sure one isn't missed.

Would be nice if Go compiler complained that the field wasn't initialized.